Update verisure documentation for 2fa bug (#17013)
home-assistant/home-assistant.github.io: source/_integrations/verisure.markdown

  type: string
{% endconfiguration %}

## 2 Factor Authentication Prerequisite

> Verisure added 2FA rules to Verisure My Pages that aren't supported through their third party API integration. If you have 2FA enabled (which is forced as default) You might not be able to use this integration. Here is the suggested way to deactivate 2FA if it's allowed in your region.

Reviewer: ```suggestion
Verisure added 2FA rules to Verisure My Pages that aren't supported through their third-party API integration. If you have 2FA enabled, which is forced by default, you might not be able to use this integration. Here is the suggested way to deactivate 2FA (if it's allowed in your region).
```

You can deactivate 2FA for your admin account and use that for Home Assistant but this isn't recommended. The steps below sets up a specific Home Assistant user and gives it restricted access.

> 1. Log in to Verisure My Pages as your admin user and create a new admin user for Home Assistant.
> 2. Log in in as your newly created Home Assistant user, you'll be prompted to set up 2FA, do that and then log out. This will make sure the options below are available.
> 3. Log in as the Home Assistant user, browse to Account and subscription -> Account -> Login Credentials -> Disable 2FA.<div class='note warning'>This will only be available if the user is admin and has logged in once with 2FA, logged out and in again.</div>
> 4. Log in as your administrator again and change the Home Assistant user to a restricted user.
> 5. Change Home Assistant Verisure config to the new user credentials in Home Assistant.
> 6. Restart Home Assistant.

Reviewer: ```suggestion
1. Log in to Verisure My Pages as your admin user and create a new admin user for Home Assistant.
2. Log in as your newly created Home Assistant user, you'll be prompted to set up 2FA, do that and then log out. This will make sure the options below are available.
3. Log in as the Home Assistant user, browse to Account and subscription -> Account -> Login Credentials -> Disable 2FA.<div class='note warning'>This will only be available if the user is admin and has logged in once with 2FA, logged out and in again.</div>
4. Log in as your administrator again and change the Home Assistant user to a restricted user.
5. Change Home Assistant Verisure config to the new user credentials in Home Assistant.
6. Restart Home Assistant.
```

> 6. Restart Home Assistant.

Reviewer: The good news it... as of 2021.4, that isn't needed anymore 😉  (the restart that is)

## 2 Factor Authentication Prerequisite

Verisure added 2FA rules to Verisure mypages that aren't supported through their third party API integration. If you have 2FA enabled (which is forced as default) You might not be able to use this integration. Here is the suggested way to deactivate 2FA if it's allowed in your region.

You can deactivate 2FA for your admin account and use that for Home Assistant but this isn't recommended. The steps below sets up a specific Home Assistant user and gives it restricted access.

> 1. `Login` to Verisure mypages as your admin user and create a new admin user for Home Assistant.

Reviewer: "Login" is not code, and should not be code fenced. Also, it is an action, thus "Log in"

```suggestion
1. Log in to Verisure My Pages as your admin user and create a new admin user for Home Assistant.
```

Author: yeah sorry about this one 🙈 
This was fixed in the latest commit

| disable_autolock | Disables autolock function for a specific lock. |
| enable_autolock | Enables autolock function for a specific lock. |
| smartcam_capture | Capture a new image from a specific smartcam. |

> ## 2 Factor Authentication Workaround

Reviewer: This is a prerequisite, let's put it above the configuration section and call it as such.

Author: I've moved it now and renamed the title. Do you think the location and new title is ok?

> In March 2021 Verisure added new 2fa rules that aren't fully support through their third party API integration. If you have 2FA enabled (which is forced as default) You might not be able to use this integration. Here are some steps for a work around.

Reviewer: Let's not date stuff, that becomes (funny enough) outdated really quick. Also, this implies a supported API, which it is not.

Also, this is not a workaround, but the advised way to do things. Verisure informed people about that as well.

> 1. Create a specific home assistant user. (not necessary but it's not recommended to turn of 2fa for your admin user).

Reviewer: - Home Assistant is spelled wrongly.
- Write an introduction that explains the separate account (instead of a note in parenthesis)

> 2. Make sure that user is admin, you've logged in & setup 2fa

Reviewer: - How does one become an admin?
- `2FA` must be capitalized

> 3. Login as the home assistant user, browse to Account and subscription -> Account -> Login Credentials -> Disable 2fa

Reviewer: - Home Assistant is spelled wrongly.
- `2FA` must be capitalized.
- 2FA is not yet available (or available to disable) on all accounts yet. So this will not work for everybody.

>    **Note this will only be available if the user is admin and has logged in once with 2fa, logged out and in again**

Reviewer: How to add notes to our documentation:

<https://developers.home-assistant.io/docs/documenting/create-page#html>

Author: Changed it to the correct format now. Thanks for sharing this, I missed this section on notes.
I had to do it inline to stop any markdown linting errors. Is this ok?

> 4. Login as your administrator again and change the home assistant user to a restricted user

Reviewer: - Home Assistant is spelled wrongly.
- `Log in` as this is an action one takes.

> 5. Change home assistant verisure config to the new user credentials in home assistant

Reviewer: Verisure and Home Assistant are spelled wrongly.

> 6. Restart home assistant

Reviewer: Home Assistant is spelled wrongly.
